crates/project-model/src/project_json.rs

```diff
@@ -93,31 +94,81 @@ pub struct Crate {
     pub build: Option<Build>,
 }
 
-/// Additional metadata about a crate, used to configure runnables.
+/// Additional, build-specific data about a crate.
 #[derive(Clone, Debug, Eq, PartialEq)]
 pub struct Build {
-    /// The name associated with this crate, according to the custom
-    /// build system being used.
+    /// The name associated with this crate.
+    ///
+    /// This is determined by the build system that produced
+    /// the `rust-project.json` in question. For instance, if buck were used,
+    /// the label might be something like `//ide/rust/rust-analyzer:rust-analyzer`.
+    ///
+    /// Do not attempt to parse the contents of this string; it is a build system-specific
+    /// identifier similar to [Crate::display_name].
```

Comment:
Nit: Crate::display_name will only show up as monospace in the docs if you wrap it in backquotes.

Reply:
thanks! fixed.